app/main.py

```diff
@@ -1 +1,37 @@
-# write your code here
+from __future__ import annotations
+
+
+class Animal:
+    alive: list["Animal"] = []
+
+    def __init__(self, name: str, helth: int = 100) -> None:
```

Comment:
There's a typo in the parameter name `helth`. It should be `health` to match the attribute name used in the class.